Python/README.md

```diff
@@ -5,9 +5,9 @@
 
 Required software:
   1. Compilers:
-  2. Linters:
+  2. Linters: [flake8](http://flake8.pycqa.org/), [pyflakes](https://github.com/PyCQA/pyflakes), [mypy](http://mypy-lang.org/) *(experimental)*
```

Comment:
@anxolerd I wouldn't say that mypy is a linter. Linters only check the code style while **static type checkers** for dynamically typed languages present a completely different sort of development tools that is certainly missing in the list. I'd prefer to add such entry for Python and JavaScript, with the examples of mypy and Flow respectively.
@tshemsedinov, what do you think about it?